src/schemas-tests/pa-yaml/v3.0/Examples/Src/Screens/Screen1.pa.yaml

```diff
@@ -1,5 +1,6 @@
 Screens:
   Screen1:
+    DisplayOrder: 1
```

Comment:
when adding new keywords to the OM, please also add to an appropriate file in the the `FullSchemaUses` folder.